### PR TITLE
config(ticdc): add large message handle option config

### LINKED ISSUE #9346: add large message handle option config

### PR DESCRIPTION
**Pull request by 3AceShowHand**
### What problem does this PR solve?

Issue Number: close #9346 

### What is changed and how it works?

* add changefeed level replica config to control how to handle large messages

#### Tests 

 - Unit test
 - Integration test

### Release note 

```release-note
`None`
```

### CODE REVIEW

File: pkg/config/sink.go [hunk 1]
```diff
@@ -141,8 +141,8 @@ type SinkConfig struct {
 
 	// DeleteOnlyOutputHandleKeyColumns is only available when the downstream is MQ.
 	DeleteOnlyOutputHandleKeyColumns *bool `toml:"delete-only-output-handle-key-columns" json:"delete-only-output-handle-key-columns,omitempty"`
-	// LargeMessageOnlyHandleKeyColumns is only available when the downstream is MQ.
-	LargeMessageOnlyHandleKeyColumns *bool `toml:"large-message-only-handle-key-columns" json:"large-message-only-handle-key-columns,omitempty"`
+	// LargeMessageHandle is only available when the downstream is MQ.
```
Comment (CharlesCheung96) [inline pkg/config/sink.go:144]: Would this configuration be more appropriate to add to KafkaConfig?

**Review by asddongmen — APPROVED** [review]

**Review by CharlesCheung96 — APPROVED** [review]

**Pull request merged**